server/Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM python:3.11.3-alpine as base
 ENV VIRTUAL_ENV=/server.env
 ENV PATH=$VIRTUAL_ENV/bin:$PATH
 
+COPY --from=gcr.io/projectsigstore/cosign:v2.1.1 /ko-app/cosign /usr/local/bin/cosign
```

Comment:
issue is when you download a key generated by latest version of cosign but attempt to use it with older version of cosign. hence explicitly pinning the test server to use older version so it can generate compatible keys with everything else which consumes it

no need to pin on chalk side as we want to test with latest cosign